README.md

```diff
@@ -503,6 +504,23 @@ List of supported Cloudflare services:
   * `cloudflare_zone`
   * `cloudflare_record`
 
+
+### Use with Logz.io
+
+Example:
+
+```
+ LOGZIO_API_TOKEN=foobar LOGZIO_BASE_URL=https://api-eu.logz.io ./terraformer import logzio -r=alerts,alert_notification_endpoints // Import Logz.io alerts and alert notification endpoints
+```
+
+List of supported Logz.io resources:
+
+* `alerts`
+    * `logzio_alert`
+* `alert notification endpoint`
+    * `logzio_endpoint`
```

Comment:
I see provider support users too. Any plans add users to terraformer?

Reply:
Yes, we haven't tested that yet as the user resource support is relatively new in the provider. Will add it some time in the future.